ci/tsan

```diff
@@ -31,3 +31,7 @@ race:crossbeam_deque*steal
 race:Backup::next_sleeper
 race:Backup::set_next_sleeper
 race:WorkerEntry::set_next_sleeper
+
+# This ignores a false positive caused by `thread::park()`/`thread::unpark()`.
```

Comment:
How would you know it's a false positive?

Reply:
But `race:pthread_cond_destroy` syntax silents **all** tsan errors for pthread_cond_destroy, isn't it?

Reply:
True. I'm not sure if we can suppress a more specific set of cases.